draft-ietf-trans-rfc6962-bis.md

```diff
@@ -690,6 +688,30 @@ Future versions of this protocol may reuse `VersionedTransType` values defined
 in this document as long as the corresponding data structures are not modified,
 and may add new `VersionedTransType` values for new or modified data structures.
 
+## Log Artifact Extensions
+
+~~~~~~~~~~~
+    enum {
+        reserved(65535)
+    } ExtensionType;
+
+    struct {
+        ExtensionType extension_type;
+        opaque extension_data<0..2^16-1>;
+    } Extension;
+~~~~~~~~~~~
+
+The `Extension` structure provides a generic extensibility for log entries,
```

Comment:
It doesn't say that SCTs and STHs are log entries.  "log entries" is the first of three items in a list; the second item is SCTs, and the third item is STHs.

However, to make it clear, I'll do "s/entries,/artifacts, including/".